README.md

```diff
@@ -50,8 +50,8 @@ To contribute to the development of xDSL follow the subsequent steps.
 
 ```bash
 git clone https://github.com/xdslproject/xdsl.git
-pip install -e .
-pip install -e .[extras]
+pip install --editable .
+pip install --requirement requirements-optional.txt
```

Comment:
This second line now only installs optional and not the basic reqs

Reply:
I just checked, the first line installs the requirements for me.